📕🔧：make refresh of hypervisor types pictogram (incremental) (#25)
OpenINF/docker-fisher: doc/glossary.md

<div align="center"><br /><br />
<figure>
  <a href="https://www.researchgate.net/figure/Type-1-and-type-2-hypervisors_fig2_335866538">

>   <img src="./img/hypervisor_types.png" alt="Type 1 and type 2 hypervisors"/>

Author: ```suggestion

<img src="./img/hypervisor_types.png" alt="Type 1 and type 2 hypervisors"/>

```

<div align="center"><br /><br />
<figure>
  <a href="https://www.researchgate.net/figure/Type-1-and-type-2-hypervisors_fig2_335866538">

<img src="./img/hypervisor_types.png" alt="Type 1 and type 2 hypervisors"/>

  </a><br />

>   <figcaption>Type 1 and type 2 hypervisors</figcaption>

Author: ```suggestion
  <figcaption>Depiction representing hypervisors of type one and two</figcaption>
```

<div align="center"><br /><br />
<figure>
  <a href="https://www.researchgate.net/figure/Type-1-and-type-2-hypervisors_fig2_335866538">

<img src="./img/hypervisor_types.png" alt="Type 1 and type 2 hypervisors"/>

  </a><br />

>   <figcaption>Depiction representing hypervisors of type one and two</figcaption>

Author: ```suggestion
  <figcaption>Fig. | Depiction of hypervisors type one and two.</figcaption>
```

<div align="center"><br /><br />
<figure>
  <a href="https://www.researchgate.net/figure/Type-1-and-type-2-hypervisors_fig2_335866538">

<img src="./img/hypervisor_types.png" alt="Type 1 and type 2 hypervisors"/>

  </a><br />
  <figcaption>
  

> Fig. | Depiction of hypervisors type one and two.

Author: ```suggestion
**Fig. | _Depiction of hypervisors type one and two._**
```
